assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -46,15 +46,15 @@ import objectWithout from '../../../../utils/objectWithout';
  *
  * @param {Object} state Current state
  * @param {Object} payload Action payload
- * @param {string} payload.firstId Element to take properties from
  * @param {string} payload.firstElement Element with properties to merge
  * @param {string} payload.secondId Element to add properties to
  * @return {Object} New state
  */
-function combineElements(state, { firstId, firstElement, secondId }) {
-  if ((!firstId && !firstElement) || !secondId) {
+function combineElements(state, { firstElement, secondId }) {
+  if (!firstElement || !secondId) {
     return state;
   }
+  const firstId = firstElement.id;
 
   const pageIndex = state.pages.findIndex(({ id }) => id === state.current);
   const page = state.pages[pageIndex];
```

Comment:
So, two lines below here, shouldn't `element` always just be `firstElement`?

Otherwise, there's easily an issue where the passed element has an ID of an existing element, and then the passed element is ignored? That was the whole purpose of either using ID or element before, but rewriting to only using full element now.

The ID should be used to find which old element is to be removed if any, but nothing more than that.

(can't comment on lines too far from lines changed in this PR, unfortunately).

Reply:
OK, changing -- I assumed for some reason that incorrect ID from the first element should abort the mission, however, apparently it's not doing it anyway currently either.